app/javascript/components/supplemental_application/sections/PreferencesTable.js

```diff
@@ -70,19 +83,21 @@ const expanderAction = (row, expanded, expandedRowToggler) => {
   <ExpanderButton label='Edit' onClick={expandedRowToggler} />)
 }
 
-const PreferencesTable = ({ application, fileBaseUrl, onSave }) => {
-  const { preferences } = application
-  const proofFiles = application.proof_files
+const PreferencesTable = ({ application, fileBaseUrl, onDismissError, onSave, onPanelClose }) => {
   const applicationMembers = getFullHousehold(application)
-  const rows = map(onlyValid(preferences), buildRow(proofFiles, fileBaseUrl))
-  return (<TableWrapper>
-    <ExpandableTable
-      columns={columns}
-      rows={rows}
-      expanderRenderer={expanderAction}
-      expandedRowRenderer={expandedRowRenderer(application, applicationMembers, onSave)}
-    />
-  </TableWrapper>)
+  const rows = buildRows(application, fileBaseUrl)
+  return (
+    <React.Fragment>
```

Comment:
@fddayan Do we need a Fragment here? Looks like it's wrapping just a single item.

Reply:
we do not